modules/userId/index.js

```diff
@@ -83,8 +83,8 @@
  * @property {(string|undefined)} publisherId - the unique identifier of the publisher in question
  * @property {(string|undefined)} ajaxTimeout - the number of milliseconds a resolution request can take before automatically being terminated
  * @property {(array|undefined)} identifiersToResolve - the identifiers from either ls|cookie to be attached to the getId query
- * @property {(string|undefined)} providedIdentifierName - defines the name of an identifier that can be found in local storage or in the cookie jar that can be sent along with the getId request. This parameter should be used whenever a customer is able to provide the most stable identifier possible
  * @property {(LiveIntentCollectConfig|undefined)} liCollectConfig - the config for LiveIntent's collect requests
+ * @property {(string|undefined)} emailHash - the email hash of the user
```

Comment:
`If provided, the hashed email address of a user`

Reply:
https://github.com/LiveIntent/Prebid.js/pull/3/commits/c7b60e999279bc7d537379c68466826b85ca293e#diff-317587762f02dab8cdc60bf20a18bba1R87